tests/vmi_configuration_test.go

```diff
@@ -3156,6 +3156,12 @@ func WithSchedulerName(schedulerName string) libvmi.Option {
 	}
 }
 
+func WithResourceProcessorLimit(cores, threads, sockets uint32) libvmi.Option {
```

Comment:
Looks a bit off placing this in another test file. Place it in `tests/libvmi/cpu.go`? And maybe rename it to `WithCPUCount`?